### ISSUE TITLE
Fixed Widths on Small/Mini Buttons in Forms

### ISSUE THREAD

**Issue by loganfranken** [ucla-oarc-web/WebBlocks#279]
I noticed that, when I was applying size modifier classes to inputs within forms (`.small`, `.mini`, and `.large`), it was applying a fixed width to those buttons on the largest viewport size. I wasn't sure if there was a reason for this? It would seem to make more sense/be more consistent to have those be `width: auto` so that they don't cut off text.

**Comment by ebollens** [comment]
This is unintentional.

Is this in the core adapter, Bootstrap adapter, or both?

I'm marking as a bug to resolve in 0.1.07.

**Comment by loganfranken** [comment]
I personally saw this in the Bootstrap Adapter.

**Comment by ebollens** [comment]
Can you please explain this better. I've now taken a look and then seem to be behaving as intended.

**Comment by loganfranken** [comment]
This could be the intended behavior; it occurs whenever the button is within a `.control` element:

``` html
    <h2>Within .control</h2>

    <form class="form">
        <div class="control">
            <input type="submit" value="This is a Really Long Button Name" class="button">
        </div>
        <div class="control">
            <input type="submit" value="This is a Really Long Button Name" class="mini button">
        </div>
        <div class="control">
            <input type="submit" value="This is a Really Long Button Name" class="small button">
        </div>
        <div class="control">
            <input type="submit" value="This is a Really Long Button Name" class="large button">
        </div>
    </form>

    <h2>Outside .control</h2>

    <input type="submit" value="This is a Really Long Button Name" class="button">
    <input type="submit" value="This is a Really Long Button Name" class="mini button">
    <input type="submit" value="This is a Really Long Button Name" class="small button">
    <input type="submit" value="This is a Really Long Button Name" class="large button">
```

(The buttons with size modifier classes within the form will have cut-off text)

**Comment by ebollens** [comment]
This is indeed a problem with the Bootstrap adapter, namely that Bootstrap is being overly imposing.

The fix is as follows:

``` css
@mixin -entity-button-mini {
    @extend .btn-mini;
    width: auto;
}

@mixin -entity-button-small {
    @extend .btn-small;
    width: auto;
}

@mixin -entity-button-large {
    @extend .btn-large;
    width: auto;
}
```

Pull request coming shortly.

**Comment by ebollens** [comment]
This ended up not being merged, because it was deprecated by #401 - which resolved the issue by getting rid of those mixins altogether within the Bootstrap adapter and they can inherit from the core adapter.
